test/reactor/create_test.exs

```diff
@@ -0,0 +1,59 @@
+defmodule Ash.Test.ReactorCreateTest do
+  @moduledoc false
+  use ExUnit.Case, async: true
+
+  defmodule Post do
+    @moduledoc false
+    use Ash.Resource, data_layer: Ash.DataLayer.Ets
+
+    ets do
+      private? true
+    end
+
+    attributes do
+      uuid_primary_key :id
+      attribute :title, :string, allow_nil?: false
+      attribute :sub_title, :string
+    end
+
+    actions do
+      defaults [:create, :read, :update, :destroy]
+    end
+  end
+
+  defmodule Api do
+    @moduledoc false
+    use Ash.Api
+
+    resources do
+      resource Post
+    end
+  end
+
+  defmodule CreatePostReactor do
+    @moduledoc false
+    use Reactor, extensions: [Ash.Reactor]
+
+    ash_reactor do
+      api Api
+    end
+
+    input :title
+    input :sub_title
+
+    create :create_post, Post, :create do
+      attribute :title, input(:title)
```

Comment:
> Since actions have attributes and arguments, I'm not sure if `attribute/2` is the base name for this constructor. What do you think?

There is both `attribute` and `argument` entities which validate against the action's defined input.

> I also think maybe having them construct the entire input might be better in the long run, something like:

I plan to solve for this more generally in Reactor by copying the `computed` template concept from Igniter.

Reply:
You don't generally want to make the caller distinguish between attributes and arguments. They should just provide "inputs" and if it writes to an attribute or is an argument value, they shouldn't have to care.

Reply:
good point.